WooCommerce/WooCommerceTests/ViewRelated/Products/ProductDetailsFactoryTests.swift

```diff
@@ -6,36 +6,14 @@ import Yosemite
 final class ProductDetailsFactoryTests: XCTestCase {
     // MARK: Simple product type
 
-    func test_factory_creates_product_form_for_simple_product_when_products_release_3_is_on() {
+    func test_factory_creates_product_form_for_simple_product() {
```

Comment:
The `expectation` below (`Wait for loading Products feature switch from app settings`) should be updated as well